README.md

```diff
@@ -1,2 +1,84 @@
-# ansible-nginx-setup
-Ansible Nginx installation repository
+# Ansible Playbook: Install Nginx on Ubuntu Local & Remote Machine
+This Ansible playbook installs and configures Nginx on an Ubuntu local & Remote Machine.
+
+# Project Structure
+
+1. `ansible.cfg` is the configuration file for Ansible. It contains settings such as the location of the inventory file and SSH options.
+2. `inventory/host.ini` is the inventory file that defines the hosts that Ansible will manage. It lists the IP addresses or domain names of the hosts and groups them into categories.
+3. `roles/` is a directory that contains all the roles for the Ansible project. In this project, there's only one role named nginx.
+4. `site.yml` is the main playbook that will be executed for the entire Ansible project. It includes the nginx role.
+5. `group_vars` directory to store environment-specific variable values for your inventory groups. This can be useful for defining variables that are specific to certain environments, such as development, staging, and production.
+
+
+
+
+# Requirements
+1. Ansible 2.10 or later
+2. Ubuntu operating system (tested on Ubuntu 20.04)
+3. Role Variables- You can modify these Role variables in the vars/main.yml file to customize the Nginx installation.
+
+# Dependencies
+NA
+
+# Usage
+
+1. Clone the repository to your local machine:
+
+```bash
+git clone https://github.com/bikashamdocs/Ansible_Repo_Final.git
+```
+2. Change directory to the cloned repository:
+
+```bash
+cd my-playbook
```

Comment:
Why is still required to be inside the specific playbook to run it, please update it as discussed ?

Reply:
We have updated the playbook to enable execution from any directory by providing the full path to the playbook.